bson/datetime_ms.py

```diff
@@ -23,10 +23,12 @@
 from typing import Any, Union, cast
 
 from bson.codec_options import DEFAULT_CODEC_OPTIONS, CodecOptions, DatetimeConversion
+from bson.errors import InvalidBSON
 from bson.tz_util import utc
 
 EPOCH_AWARE = datetime.datetime.fromtimestamp(0, utc)
 EPOCH_NAIVE = EPOCH_AWARE.replace(tzinfo=None)
+DATETIME_ERROR_SUGGESTION = "(Use CodecOptions(datetime_conversion=DATETIME_AUTO) or MongoClient(datetime_conversion='DATETIME_AUTO'))"
```

Comment:
I suggest "(Consider using Codec...)" because not all users will want/need to use this feature when this error happens. For example sometimes the user accidentally introduced out of range years and they might want to remove those dates instead of preserving them. 

Also it would be great to link to `https://pymongo.readthedocs.io/en/stable/examples/datetimes.html#handling-out-of-range-datetimes`.

One last thing is that `DATETIME_ERROR_SUGGESTION` should be `_DATETIME_ERROR_SUGGESTION` to make it private. Anything not beginning with `_` can be misinterpreted as public api.